packages/maskbook/src/plugins/Trader/constants/trader.ts

```diff
@@ -46,6 +46,28 @@ export const TRADE_CONSTANTS = {
         [ChainId.Kovan]: '',
         [ChainId.Gorli]: '',
     },
+    BALANCER_ETH_ADDRESS: {
+        [ChainId.Mainnet]: '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE',
```

Comment:
?

Reply:
This is an imaginary address (just like 0x0....0). In addition, It's checksummed.